src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpConstructorSnippetCompletionProviderTests.cs

```diff
@@ -168,5 +168,106 @@ public MyClass1()
 }";
             await VerifyCustomCommitProviderAsync(markupBeforeCommit, ItemToCommit, expectedCodeAfterCommit);
         }
+
+        [WpfTheory]
+        [InlineData("class")]
+        [InlineData("enum")]
+        [InlineData("interface")]
+        [InlineData("record")]
+        [InlineData("struct")]
+        [InlineData("record class")]
+        [InlineData("record struct")]
+        [Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InsertConstructorSnippetHasNestedTypeTest(string keyword)
+        {
+            var markupBeforeCommit =
+@"class Outer
+{
+    $$
+    %keyword% Inner
+    {
+    }
+}".Replace("%keyword%", keyword);
```

Comment:
Use string interpolation in all tests. This `%keyword%` notation is really confusing

Reply:
@DoctorKrolic , Should I change to Interpolation?

``` csharp
$@"class Outer
{{
    $$
    { keyword } Inner
    {{
    }}
}}";
```

Reply:
Yes, I think it looks if not better then at least more familiar. If you want to improve it even more use raw string literals (I would preffer this).

Reply:
@DoctorKrolic I pushed modified code.

``` csharp
$$"""
class Outer
{
    $$
    {{keyword}} Inner
    {
    }
}
""";
```